pkg/e2e/e2e.go

```diff
@@ -743,7 +743,7 @@ func runTestsInPhase(phase string, description string) bool {
 		}
 		clusterState = cluster.State()
 	}
-	if !viper.GetBool(config.DryRun) && clusterState == spi.ClusterStateReady {
```

Comment:
Any motivation for this being replaced in favour if it being an additional argument to the function? It could be left as it is and still behave the same.

Reply:
there are multiple sources of truth for this var. We add it to the ginkgo config explicity so we should refer to that here and elsewhere.

Reply:
Fair enough, that makes sense